pkg/driver/node_server.go

```diff
@@ -1852,6 +1860,13 @@ func (driver *Driver) nodeGetInfo() (string, error) {
 				if initiator.Chap != nil {
 					chapUsername = initiator.Chap.Name
 					chapPassword = initiator.Chap.Password
+				} else {
```

Comment:
I think this causes a mismatch if something is already set in iscsid.conf and provided values with csi-driver are different. With this version, can we ignore iscsid.conf and always use configured values during controller publish and node stage?

Reply:
i had kept this for backward compatibility but agree iscsid.conf can be ignored to keep it simple.

Reply:
We aren't supporting different CHAP users from different nodes are we?

Reply:
👍